card/lib/card/tasks/card/migrate.rake

```diff
@@ -39,20 +44,21 @@ namespace :card do
     desc "migrate structure"
     task structure: :environment do
       ENV["SCHEMA"] ||= "#{Cardio.gem_root}/db/schema.rb"
-      ActiveRecord::Base.dump_schema_after_migration = false
-      Cardio.schema_mode(:structure) do |paths|
-        ActiveRecord::Migrator.migrations_paths = paths
-        ActiveRecord::Migrator.migrate paths, version
-        Rake::Task["db:_dump"].invoke # write schema.rb
+      without_dumping do
+        Cardio.schema_mode(:structure) do |paths|
+          ActiveRecord::Migrator.migrations_paths = paths
+          ActiveRecord::Migrator.migrate paths, version
+        end
```

Comment:
how do we update schema now?

Reply:
Maybe we should add our own task for it. I don't think I will remember the  "db:_dump" task in the far future.